content/applications/general/payment_acquirers/adyen.rst

```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
+accept **online** and **point-of-sale** payments.
+
+.. image:: media/adyen_logo.png
+   :align: center
+   :alt: Adyen logo
+
+This Payment Acquirer offers additional options that are not available for other :doc:`Payment
+Acquirers <../payment_acquirers>`, such as the ability to refund partially a payment directly from
+Odoo, to capture the payment after the delivery, or to link a point of sale's terminal.
+
+Payment Acquirer Configuration
+==============================
+
+If not done yet, `contact Adyen <https://www.adyen.com/contact/sales>`__ to get your live account.
+
+.. seealso::
+
+   - `Get started with Adyen
+     <https://docs.adyen.com/get-started-with-adyen>`__
+
+Credential
+----------
+
+Odoo needs your **API Credentials** to connect with your Adyen account, which comprise:
+
+- Merchant Account
+- API Key
+- HMAC Key
+- Checkout API URL
+- Payment API URL
+- Recurring API URL
+
+To retrieve the API Key, log into your Adyen account, go to :menuselection:`Account --> API
+Credentials --> Your username --> Authentication`, and generate your **API Key**.
+
+.. image:: media/adyen_api_key.png
+   :align: center
+   :alt: Generate a new API Key.
+
+To retrieve the HMAC Key, TODO
```

Comment:
TODO :D

Reply:
I said it before we made the review!  =D